src/App.js

```diff
@@ -1,23 +1,37 @@
-import logo from './logo.svg';
-import './App.css';
+import React from "react";
+import Header from "./components/header/Header";
+import Main from "./components/main/Main";
+import Footer from "./components/footer/Footer";
+import data from "./data.json";
 
 function App() {
+  console.log(data);
```

Comment:
Please don't forget to remove `console.log`s before merging.

Reply:
ich habe entfernt.